builder/src/permissioned.rs

```diff
@@ -385,7 +387,8 @@ pub async fn init_hotshot<
             .unwrap(),
         ConsensusMetricsValue::new(metrics),
         da_storage,
-        TestAuctionResultsProvider::default(),
+        // TODO Passing dummy url for now.
+        SolverAuctionResultsProvider(Url::from_str("http://some.url").unwrap()),
```

Comment:
Does builder need to be able to fetch auction results?

Reply:
I think that *this builder* does not. marketplace_builder might need this though.